portfolio/src/main/webapp/contact.html

```diff
@@ -8,16 +8,17 @@
   </head>
   
   <body class="background">
-    <button onclick=goToContactPage()><b>Contact</b></button>
-    <button><b>Projects</b></button>
-    <button><b>About</b></button>
-    <button onclick=goToHomePage()><b>Home</b></button>
+    <div class="menu">  
+        <a href='index.html'><b>Home</b></a>
+        <a><b>About</b></a>
+        <a><b>Projects</b></a>
+        <a href='contact.html'><b>Contact</b></a>
```

Comment:
FYI - <b> can be replaced by font styles (css) e.g. make everything inside div classed menu use bold.